all/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.10-slim as build
+FROM python:3.11.0-slim as build
```

Comment:
Not sure why it included the patch level here, but upgrading from 3.10->3.11 sounds good.